src/components/UniversalNavbarExpanded/DropdownNav/DropdownChildren.module.scss

```diff
@@ -0,0 +1,94 @@
+@import '../../Media/Media.scss';
+@import '../../Variables.scss';
+
+$laptop-range-medium-end: 1050px;
+
+.dropdownNavChildren {
+  display: none;
+  position: absolute;
+  top: var(--Space-64);
+  left: 0;
+  width: 100%;
+  background: var(--White);
+  border-bottom: $navbar-border;
+  min-height: 220px;
+}
+
+.dropdownNavChild {
+  margin-bottom: var(--Space-16);
+  a {
+    text-decoration: none;
+    &:hover {
+      text-decoration: none;
+      opacity: 0.78;
+      span {
+        text-decoration: underline;
+      }
+    }
+  }
+}
+
+.dropdownNavChildTextIcon {
+  display: inline-block;
+}
+
+.dropdownNavChildrenInner {
+  display: flex;
+  padding: var(--Space-32) 0 var(--Space-24);
+}
+
+.dropdownNavChildrenCta {
+  margin-right: 120px;
+
+  &:hover {
+    span {
+      opacity: 0.78;
```

Comment:
yeah, so we could have all of these do:
```css
opacity: var(--opacity);
```

or similar

Reply:
dd7d058